cmd/karavictl/cmd/storage_create.go

```diff
@@ -277,16 +367,12 @@ func init() {
 	if err != nil {
 		reportErrorAndExit(JSONOutput, storageCreateCmd.ErrOrStderr(), err)
 	}
-	storageCreateCmd.Flags().StringP("system-id", "s", "", "System identifier")
-	err = storageCreateCmd.MarkFlagRequired("system-id")
-	if err != nil {
-		reportErrorAndExit(JSONOutput, storageCreateCmd.ErrOrStderr(), err)
-	}
 	storageCreateCmd.Flags().StringP("user", "u", "", "Username")
 	err = storageCreateCmd.MarkFlagRequired("user")
 	if err != nil {
 		reportErrorAndExit(JSONOutput, storageCreateCmd.ErrOrStderr(), err)
 	}
+	storageCreateCmd.Flags().StringSliceP("system-id", "s", []string{""}, "System identifier")
```

Comment:
this should be a required flag storageCreateCmd.MarkFlagRequired("system-id")

Reply:
For registering all PowerMax arrays in Unisphere, it's not required.